spec/support/factory_bot.rb

```diff
@@ -0,0 +1,5 @@
+require 'factory_bot_rails'
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
+  FactoryBot.definition_file_paths = [File.expand_path('factories', __dir__)]
```

Comment:
you need this only
> If you're using a separate directory structure for your factories
([GETTING_STARTED](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#using-without-bundler))